fastlane/Fastfile

```diff
@@ -932,18 +943,24 @@ platform :ios do
     match(
       type: 'enterprise',
       team_id: get_required_env('INT_EXPORT_TEAM_ID'),
-      readonly: true,
-      app_identifier: ALPHA_BUNDLE_IDENTIFIERS
+      app_identifier: ALPHA_BUNDLE_IDENTIFIERS,
+      readonly: true
     )
   end
 
   private_lane :appstore_code_signing do
     match(
       type: 'appstore',
       team_id: get_required_env('EXT_EXPORT_TEAM_ID'),
-      readonly: false,
       app_identifier: MAIN_BUNDLE_IDENTIFIERS,
-      template_name: 'Apple Pay Pass Suppression'
+      # For a time, we used to have this parameter specifying a entitlements.
+      # In August 2022, `match` started being unable to generate new
+      # provisioning profiles and the only solution I found was to remove it.
+      #
+      # Leaving it here for future referece as something to try if `match`
+      # gives us trouble again.
+      # template_name: 'Apple Pay Pass Suppression'
```

Comment:
I'm not sure whether more detail is needed here? I'm not keen in adding links to internal documents (even if in shorthand form) in the code. There should be detail here to search for the information internally if necessary. What do you think?

Reply:
```suggestion
      # For a time, we used to have this parameter specifying a entitlements.
      # In August 2022, `match` started being unable to generate new
      # provisioning profiles and the only solution I found was to remove it.
      #
      # Leaving it here for future reference as something to try if `match`
      # gives us trouble again.
      # template_name: 'Apple Pay Pass Suppression'
```

Reply:
At first I misread (multiple times, even… ☕ ) this as "`match` started being **able** to […]" and that sentence got me so confused, because I was like "so wait, match got better with a new feature, and you had to find a way to… fix it?". Stumbled me for two good minutes… until I was about to make a GitHub suggestion to reword this and finally notice that you wrote "unable", not "able".
I think the "starting being" wording led me astray, as I'm not used in hearing "start to be unable" and would expect a wording like "stop being able" instead.

Also I'm not sure if that wording is supposed to mean that this is a bug in `match` (it started to have a bug which made it fail to create new profiles) or a feature (they purposely removed that ability because it wasn't working as expected or whatnot, and thus removed the `template_name` parameter as well).

Your subsequent sentence suggesting that we might re-introduce the `template_name` parameter again in the future lead me to finally understand that this might probably be a bug and not a conscious decision by fastlane, but the "to try if match gives us trouble again" added to the confusion 😅  As in: if the bug was when match tried to create new profiles because of this parameter, how come match would… give us trouble again without it? If anything, I feel that the reason why we'd want to add the parameter back is if match fixed the profile creation bug and we wanted to give it back a try, no?

Reply:
The truth is... I don't know if this is a bug in `match`, a bug on Apple's side, or something that never worked, was worked around in a different way, and we only found out about it now because we decided to generate new profiles 🙃 

It could be the latter, because the [automation](https://github.com/Automattic/match-regenerator/blob/edef2ca14b0f62ccfab0f6426711daf296d56b98/fastlane/Fastfile) I cobbled together in the past to regenerate the provisioning profiles across all app never had `template_name` and worked back then. I had forgotten about that code and only though of it in writing this.

I think I'm just going to remove the comment. Git remembers.

Reply:
👍 on removing the comment. git blame ftw 😛